common/CMakeLists.txt

```diff
@@ -0,0 +1,240 @@
+PROJECT(nns_ros_bridge C CXX)
+
+## Inner functions
+FUNCTION(search_pkg_conf_dir_r_inner search_path return_list)
+    FILE(GLOB_RECURSE new_list ${search_path}/*.pc)
+    SET(dir_list "")
+    FOREACH(file_path ${new_list})
+        GET_FILENAME_COMPONENT(dir_path ${file_path} PATH)
+        SET(dir_list ${dir_list} ${dir_path})
+    ENDFOREACH()
+    LIST(REMOVE_DUPLICATES dir_list)
+    SET(${return_list} ${dir_list} PARENT_SCOPE)
+ENDFUNCTION(search_pkg_conf_dir_r_inner)
+
+FUNCTION(search_pkg_conf_dirs search_paths return_list)
+    FOREACH(search_path ${search_paths})
+        search_pkg_conf_dir_r_inner(${search_path} inner_return)
+        SET(all_list ${all_list} ${inner_return})
+    ENDFOREACH()
+    SET(${return_list} ${all_list} PARENT_SCOPE)
+ENDFUNCTION(search_pkg_conf_dirs)
+
+## Set CMAKE_PREFIX_PATH to kinect default path
```

Comment:
Yes, I'd like to have Kinect as well :smile:

Reply:
@myungjoo  Fixed. I have no idea where 'kinect' came from :stuck_out_tongue_closed_eyes: